fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadMgr.java

```diff
@@ -166,10 +167,11 @@ public void beginLoadTaskFromBackend(String dbName, String tableName, String lab
         StreamLoadTask task = null;
         Database db = checkDbName(dbName);
         long dbId = db.getId();
+        Table table = checkMeta(db, tableName);
 
         writeLock();
         try {
-            task = createLoadTask(db, tableName, label, user, clientIp, timeoutMillis, isRoutineLoad, warehouseId);
+            task = createLoadTask(db, table, label, user, clientIp, timeoutMillis, isRoutineLoad, warehouseId);
```

Comment:
it is a little weird that the `addLoadTask` is public accessible and inside the body it is not protected by lock. throw the data integrity responsibility to the caller.

Reply:
task creation can be out of writeLoad, only `addLoadTask()` needs to be put under writeload protection.

Reply:
the first and third steps should be under writeload protection

1. task = idToStreamLoadTask.get(label); if (task != null) {...}
2. createLoadTask
3. addLoadTask